providers/workflow-es-mysql/package.json

```diff
@@ -22,11 +22,12 @@
   "devDependencies": {
     "@types/jasmine": "^3.3.1",
     "@types/node": "^10.12.12",
-    "jasmine": "^3.3.1",
-    "jasmine-core": "^3.3.0",
+    "jasmine": "^2.5.2",
+    "jasmine-core": "^2.4.1",
```

Comment:
why did you need to downgrade jasmine?

Reply:
It was a test to see if this was the problem with the docker auth. I forgot to upgrade it.